src/apps/auth-login/Login.tsx

```diff
@@ -1,85 +1,136 @@
-import { useEffect, useState } from 'react';
-import type { Session } from '@supabase/supabase-js';
-import { supabase } from '@backend/supabaseBrowserClient';
-import { isLoggedIn } from '@backend/auth';
-import type { Translations } from 'src/Types';
-import { StateChecking, StateLoginForm, StateMagicLink } from './states';
+import { useEffect, useState } from "react";
+import type { Session } from "@supabase/supabase-js";
+import { supabase } from "@backend/supabaseBrowserClient";
+import { isLoggedIn } from "@backend/auth";
+import type { LoginMethod, Translations } from "src/Types";
+import { StateChecking, StateLoginForm, StateMagicLink } from "./states";
+import { LoginMethodSelector } from "@apps/auth-login/LoginMethodSelector";
 
-import './Login.css';
+import "./Login.css";
 
 const SSO_DOMAIN = import.meta.env.PUBLIC_SSO_DOMAIN;
```

Comment:
The SSO_DOMAIN env variable is now no longer used, I believe?

Reply:
I have removed the variable because the config will be required.

Reply:
Actually I have not done this yet. I will come back to it but need this in the develop branch quickly